nav2_controller/plugins.xml

```diff
@@ -0,0 +1,17 @@
+<class_libraries>
+  <library path="simple_progress_checker">
+    <class type="nav2_controller::SimpleProgressChecker" base_class_type="nav2_core::ProgressChecker">
+      <description></description>
```

Comment:
Please add descriptions for each (so people know why one over another)

Reply:
Done